mapsims/cmb.py

```diff
@@ -0,0 +1,81 @@
+import healpy as hp, numpy as np
+from pixell import enmap, curvedsky
+
+def get_cmb_sky(iteration_num, 
+                nside = None, #set this if healpix is desired
+                shape = None, #set shape and wcs if CAR maps are desired
+                wcs = None,
+                lensed = True,
+                aberrated = False,
+                pol = True,
+                cmb_set = 0, #We allow for more than one CMB map per lensing map
+                cmb_dir = None, 
+                nfreqs = 1):
+    '''
+    Return a CMB map from stored alm's.  This can be in Healpix format
+    (if nside is specified) or rectangular pixel format (if wcs and shape are
+    specified).  The lensed alm's are pre-stored.
+    If rectangular, it returns a stack of enmaps of shape (nfreqs, ncomp, ny, nx). 
```

Comment:
what do you mean by stack?

Reply:
It is a pixell.enmap object.  This allows for a stack of (nfreqs, ncomp) separate rectangular maps, where nfreqs is the number of frequencies and ncomp is typically 3 (for T, Q, U)

Reply:
Or more generally, some subclass of a numpy array with more leading dimensions than (npix,) for healpix or (Ny,Nx,) for pixell.ndmap.